src/index.ts

```diff
@@ -282,12 +282,32 @@ export async function build({
 		...(await glob('.deno/**/*', workPath)),
 	};
 
-	console.log('Detected source files:');
-
 	if (denoTsConfig) {
 		sourceFiles.add(denoTsConfig);
 	}
 
+	if(config.includeFiles) {
+
+		const includeFiles =
+		   typeof config.includeFiles === 'string'
+		  	? [config.includeFiles]
+			: config.includeFiles;
+		
+		console.log("Detected additional files");
+		for(const pattern of includeFiles) {
+			console.log(` - ${pattern}`);
+			const files = await glob(pattern, workPath);
+
+			for(const filename of Object.keys(files)) {
+				if(!sourceFiles.has(filename)) {
+					console.log(`   - ${filename}`);
+					sourceFiles.add(filename);
+				}
+			}
```

Comment:
There's not much of a reason to add these to `sourceFiles`, since the `glob()` function already returns the `FileFsRef` instances that the `outputFiles` object is expecting. Instead we can just use `Object.assign()`:

```suggestion
			Object.assign(outputFiles, await glob(pattern, workPath));
```

Reply:
Very nice. thanks!